apis/tools/env/wazuh-indexer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM opensearchproject/opensearch:2.18.0
+FROM opensearchproject/opensearch:2.19.1
```

Comment:
The new plugins containing the RBAC indices require the version 2.19+ to be installed